api/model/api.go

```diff
@@ -261,6 +262,12 @@ type Target struct {
 	Namespace string `json:"namespace"`
 }
 
+func (t *Target) Parse(v string) {
+	p := strings.Split(v, "/")
+	t.Namespace = p[0]
+	t.Name = p[1]
```

Comment:
This could be made more safe. e.g. what happens if the string doesn't contain a `/`?